examples/Vertical.tsx

```diff
@@ -0,0 +1,22 @@
+import type { ComponentType, FC } from 'react';
+import React from 'react';
+import { Slider } from '../src';
+import { Orientation } from '../src/Slider';
+
+const Vertical: FC<typeof Slider extends ComponentType<infer T> ? Omit<T, 'children'> : never> = (sliderProps) =>  (
```

Comment:
Does it support direction on different screenwidth, would be nice to see example

Reply:
Adding breakpoints would add some complexity and I don't think it will get used very often. For now it would be best solved outside the code by having the breakpoints in React. Then you can set the direction based on the breakpoints afterwards. 

```
const [isMobile, setIsMobile] = useState<boolean>(window.matchMedia('(max-width: 600px)').matches);

return (
    <Slider orientation={isMobile ? 'vertical' : 'horizontal'}>
        ...slides
    </Slider>
);
```

You'd also have to register a resize listener to check if it's still mobile on a resize.